kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -0,0 +1,45 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class FlightFinder {
+    void findFlight (Flight flight) throws RoteNotFoundException {
+        Map<String, Boolean> flights = new HashMap<>();
+
+        flights.put("Berlin", true);
+        flights.put("New York", true);
+        flights.put("London", true);
+        flights.put("Warsaw", true);
+        flights.put("Moscow", true);
```

Comment:
Mapa lotnisk nie powinna byc budowana przy kazdym wywolaniu metody. To nieefektywne. Zbuduj ja w konstruktorze